pkg/ddevapp/config.go

```diff
@@ -569,7 +569,7 @@ func (app *DdevApp) CheckDeprecations() {
 func (app *DdevApp) FixObsolete() {
 
 	// Remove old in-project commands (which have been moved to global)
-	for _, command := range []string{"db/mysql", "host/launch", "web/xdebug"} {
+	for _, command := range []string{"db/mysql", "host/launch", "web/xdebug", "host/yarn"} {
```

Comment:
This isn't exactly correct - this is removing from the project's .ddev, and we need to remove from global. So we need another stanza like this for global.

Reply:
Isn't the global commands stuff rewritten each time anyways?

Reply:
Yes, but no removal happens.

Reply:
I guess you need to help me out then, because I obviously misunderstand what you mean.
My experience is that the whole `.global_commands` folder is eliminated every time I `ddev start`
(at least the handles are gone if I have wsl open in one of those subfolders in the meantime)

Reply:
Yeah, .global_commands is not where global commands live, they're just copied in there on start/etc. They live in ~/.ddev/commands

I'm happy to add the stanza though. Have to get the global config dir and do the work there.

Reply:
Oh, I didn't know that. Thanks for explaining. I can add the go stuff then.